cmw-dto/src/main/java/com/liberologico/cloudesire/cmw/model/dto/EventDTO.java

```diff
@@ -3,23 +3,37 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.liberologico.cloudesire.cmw.model.enums.CmwEventType;
 import com.liberologico.cloudesire.cmw.model.enums.EventRecipientType;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
 
 import javax.validation.Valid;
 import java.util.Date;
 
 /**
- * This is the object exposed by the API, not the notification sent
+ * This is the object exposed by the API, not the notification sent via HTTP
  *
  * @see EventNotificationDTO
  */
+@ApiModel( "An event dispatched by the platform to a recipient" )
 public class EventDTO extends BaseEntityDTO
 {
+    @ApiModelProperty( value = "The name of the entity", example = "Subscription" )
```

Comment:
I'm not sure about this if it's lowercase too or not

Reply:
https://docs.cloudesire.com/docs/syndication.html#anatomy-of-an-event-request